### ISSUE TITLE
Installation & Running

### ISSUE THREAD

**Issue by llsjk** [TrnsltLife/HunspellXML#1]
I can't get it to run, it just show the logo and stays there.

**Comment by TrnsltLife** [comment]
Yeah, I see what you mean. It apparently doesn't run under Java 8. Is that what you have? If you have Java 6 or Java 7 installed (or if you install one) you can run it with those.

Example from Windows command prompt:
set JAVA_HOME7="C:\Program Files\Java\jdk1.7.0_51"
"%JAVA_HOME7%\bin\java.exe" -jar HunspellXML-Converter-0.23.jar

I'll have to work on updating it so it will work with Java 8.

**Comment by llsjk** [comment]
I don't use MS Windows.

My Java:
java version "1.8.0_60"
Java(TM) SE Runtime Environment (build 1.8.0_60-b27)
Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)

**Comment by joemaza** [comment]
I installed JRE 7u80 and explicitly called the binary on the .jar. It worked for me.
